scripts/ifAlias

```diff
@@ -18,8 +18,17 @@ BASE='.1.3.6.1.2.1.31.1.1.1.18'
 GET_TYPE="$1"
 GET_OID="$2"
 
+UNAME="$(/usr/bin/uname)"
+
+COUNT="$(echo "${UNAME}" | grep -ci 'bsd$')"
+if [ "${COUNT}" -eq 1 ]; then
+    UNAME="BSD"
+fi
+
 # cache ip link output
-IP_LINK="$(ip link)"
+if [ "${UNAME}" = 'Linux' ]; then
+    IP_LINK="$(ip link)"
+fi
```

Comment:
Do we have to cache this? We could thus do without a global variable.

Reply:
Results in fewer external calls.

Really should do the same with ifconfig, but that can be done some other time.